src/SegmentedControl/SegmentedControl.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import lodash from "lodash";
+
+import "./SegmentedControl.less";
+
+/*
+  SegmentedControl selection. Shows several options
+  as defined in the props and allows the user to select
+  one of those options.
+*/
+export class SegmentedControl extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {selected: props.defaultValue};
```

Comment:
react adopts a specific pattern with form controls that we might want to follow here (https://facebook.github.io/react/docs/forms.html):
`defaultValue` is only used if an explicit value hasn't been selected.

i think it's also advised not to initialize state from props, since props can change and go out of sync with the state.

in this case, you could defer the defaulting logic to the `render()` method instead and leave `state.selected = null` until it's explicitly set